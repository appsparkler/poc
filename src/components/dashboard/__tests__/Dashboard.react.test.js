import React from 'react'
import renderer from 'react-test-renderer'
import Dashboard from '../Dashboard.react'
import configureMockStore from 'redux-mock-store'
import {Provider} from 'react-redux'
// import {BrowserRouter} from 'react-router-dom'

it('should render as expected', () => {
  const mockStore = configureMockStore()
  const store = mockStore({
    project: {
      fetchingProjects: true,
      projects: [{
        id: '1',
        title: 'Title 1',
        content: 'Content...',
      }],
    },
  })
  const component = renderer.create(
      <Provider store={store}>
        <Dashboard />
      </Provider>
  )
  const tree = component.toJSON()
  expect.assertions(1)
  expect(tree).toMatchSnapshot()
})
