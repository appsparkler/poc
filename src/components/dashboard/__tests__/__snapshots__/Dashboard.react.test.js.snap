// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render as expected 1`] = `
<div
  className="dashboard container"
>
  <div
    className="row"
  >
    <div
      className="col s12 m6"
    >
      <div
        className="project-list section"
        style={
          Object {
            "maxHeight": "500px",
            "overflow": "auto",
            "padding": "0 20px",
          }
        }
      >
        <div
          className="progress"
        >
          <div
            className="indeterminate"
          />
        </div>
      </div>
    </div>
    <div
      className="col s12 m5 offset-m1"
    >
      <div>
        <p>
          Notifications
        </p>
      </div>
    </div>
  </div>
</div>
`;
